src/main/java/com/unb/tracker/TrackerController.java

```diff
@@ -1,15 +1,19 @@
 package com.unb.tracker;
 
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
 import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 
-@RestController
+@Controller
+@EnableAutoConfiguration
 public class TrackerController {
-    @RequestMapping(value="/", method = RequestMethod.GET, produces = {MediaType.TEXT_HTML_VALUE})
+    @RequestMapping(value="/", method = RequestMethod.GET)
     public String index(ModelMap map) {
-        return "<html><h1>SWE 4103 Project</h1></html>";
+        return "index";
```

Comment:
One thing I want to point out here, since I return `index`, the template engine looks for a file called `index.twig` in the templates folder!